org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/context/package-info.java

```diff
@@ -15,5 +15,5 @@
  */
 
 @org.osgi.annotation.bundle.Export
-@org.osgi.annotation.versioning.Version("1.0.0")
+@org.osgi.annotation.versioning.Version("2.0.0")
```

Comment:
We don't need to bump the package version since we are still making `0` major version bundles. If the baselining against 0.9.0 bundles is complaining, we should disable baselining in the poms until we actually have a version 1.0.0 release.

Reply:
Fixed.